content/en/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination.md

```diff
@@ -84,9 +88,16 @@ See the full [Kinesis CloudFormation template][1].
 3. Check the `Subscriptions` column in the [log groups index page][1] to confirm that the new Kinesis stream is now subscribed to your log groups.
 ​
 If you want to push logs directly to the delivery stream without going through a Kinesis data stream, you can subscribe the CloudWatch log groups directly to the Kinesis Firehose Destination by adding the Kinesis Firehose ARN in the `destination-arn` parameter of the subscription filter, as shown in [the AWS Subscription Filters documentation][4] (step 12).
-​
+
+## Searching for AWS Kinesis logs in Datadog
+
+Once you have set up an Amazon Kinesis delivery stream, logs subscribed to your delivery stream are now available for analysis within Datadog. To populate all logs by ARN:
+
+1. Navigate to [Logs Explorer][5] in Datadog to see all of your subscribed logs.
```

Comment:
```suggestion
1. Navigate to the [Logs Explorer][5] in Datadog to see all of your subscribed logs.
```

Reply:
Should this link send users to the in-app Logs Explorer or the Logs Explorer documentation?